src/pages/workspace/taxes/WorkspaceCreateTaxPage.tsx

```diff
@@ -0,0 +1,124 @@
+            if (!ValidationUtils.isValidPercentage(value)) {
+                errors[INPUT_IDS.VALUE] = 'workspace.taxes.errors.valuePercentageRange';
+            }
+
+            const name = values[INPUT_IDS.NAME];
+            if (policy?.taxRates?.taxes && ValidationUtils.isExistingTaxName(name, policy.taxRates.taxes)) {
+                errors[INPUT_IDS.NAME] = 'workspace.taxes.errors.taxRateAlreadyExists';
+            }
+
+            return errors;
+        },
+        [policy?.taxRates?.taxes],
+    );
+
+    const submitForm = useCallback(
+        ({value, ...values}: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_NEW_TAX_FORM>) => {
+            const taxRate = {
+                ...values,
+                value: getTaxValueWithPercentage(value),
+                code: getNextTaxCode(values[INPUT_IDS.NAME], policy?.taxRates?.taxes),
+            } satisfies TaxRate;
+            createPolicyTax(policyID, taxRate);
+            Navigation.goBack();
+        },
+        [policy?.taxRates?.taxes, policyID],
+    );
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={policyID}>
+                <ScreenWrapper
+                    testID={WorkspaceCreateTaxPage.displayName}
+                    includeSafeAreaPaddingBottom={false}
+                    style={[styles.defaultModalContainer]}
+                >
+                    <View style={[styles.h100, styles.flex1, styles.justifyContentBetween]}>
+                        <HeaderWithBackButton title={translate('workspace.taxes.addRate')} />
+                        <FormProvider
+                            style={[styles.flexGrow1, styles.mh5]}
+                            formID={ONYXKEYS.FORMS.WORKSPACE_NEW_TAX_FORM}
+                            onSubmit={submitForm}
+                            validate={validate}
+                            submitButtonText={translate('common.save')}
+                            enabledWhenOffline
+                            shouldValidateOnBlur={false}
+                            disablePressOnEnter={false}
+                        >
+                            <View style={styles.mhn5}>
+                                <InputWrapper
+                                    InputComponent={TextPicker}
+                                    inputID={INPUT_IDS.NAME}
+                                    label={translate('common.name')}
+                                    description={translate('common.name')}
+                                    rightLabel={translate('common.required')}
+                                    accessibilityLabel={translate('workspace.editor.nameInputLabel')}
+                                    maxLength={CONST.TAX_RATES.NAME_MAX_LENGTH}
+                                    multiline={false}
+                                    role={CONST.ROLE.PRESENTATION}
+                                    autoFocus
```

Comment:
autoFocus does work properly for all the Platforms, We should use useFocusEffect hook to focus on the input which is also suggested in the [docs](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#how-do-i-auto-focus-a-textinput-using-usefocuseffect). autoFocus cause this Issue https://github.com/Expensify/App/issues/41800 and  the keyboard does not automatically appear in the Name menu